hub/core/serialize.py

```diff
@@ -22,6 +25,21 @@
 BaseTypes = Union[np.ndarray, list, int, float, bool, np.integer, np.floating, np.bool_]
 
 
+def infer_header_num_bytes(
+    version: str, shape_info: np.ndarray, byte_positions: np.ndarray
+):
+    """Calculates the number of header bytes in a chunk without serializing it.
+
+    Args:
+        version: (str) Version of hub library
+        shape_info: (numpy.ndarray) Encoded shapes info from the chunk's `ShapeEncoder` instance.
+        byte_positions: (numpy.ndarray) Encoded byte positions from the chunk's `BytePositionsEncoder` instance.
+
+    Returns:
+        Length of the headers of chunk when serialized as int"""
+    return len(version) + shape_info.nbytes + byte_positions.nbytes + 13
```

Comment:
what 13 stands for? shouldn't be it as a constant?

Reply:
Version string length : 1 byte +
shape encoder shape: 8 bytes +
byte positions length: 4 bytes = 13 bytes